### PR TITLE
Initial import of generated files

### PR DESCRIPTION
**Pull request by kyessenov**
Signed-off-by: Kuat Yessenov <kuat@google.com>

### CODE REVIEW

**Comment by kyessenov** [comment]
Waiting for the API PR to be merged...

File: bazel/update.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```
Comment (rshriram) [inline bazel/update.sh]: Just thinking.. does it make sense to fold this into the main envoy repo, and we can reuse the lyft/envoy-build bazel image? That image has all the stuff we need. Basically, we reduce this command down to one liner (docker run -it lyft/envoy-build -c ... )

**Comment by kyessenov** [comment]
I can move them to top-level. Would need to rewrite the import for "api".
Is referring by `github.com/envoyproxy/go-control-plane/vendor/api` not good?

File: bazel/update.sh (hunk 1 above)
Comment (kyessenov) [reply bazel/update.sh]: This script just bulids the go stuff from API repo skipping cc and py code.

**Comment by davecheney** [comment]
Bad things will happen when dev or glide try’s to “collapse” vendored dependencies. Can you just move it up one level to github.com/envoyproxy/go-control-plane/api .

**Comment by kyessenov** [comment]
Done moving. The build passes since `api` is skipped over (and not pulled indirectly since there is no other code)

**Comment by kyessenov** [comment]
I'll follow up with another update (the SHA references my PR, not master)

**Pull request merged**